packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -167,7 +167,12 @@ async function _getDashboardStats(app) {
   dashboard.nPools = poolIds.size
   dashboard.nHosts = hosts.length
 
-  if (await app.hasFeatureAuthorization('LIST_MISSING_PATCHES')) {
+  const [hasFeatureAuthorizationListMissingPatches, remotesInfo] = await Promise.all([
+    app.hasFeatureAuthorization('LIST_MISSING_PATCHES'),
+    app.getAllRemotesInfo(),
```

Comment:
Why do you wait for this call here? Is it an optimization? Because this will delay the other async calls like `listMissingPatches` if it takes too long. An optimization would be to call it here and `await` it when you need it, but it might make the code harder to read so if it's not worth it, maybe just call and await where you need it?